app/question_list_helper.html

```diff
@@ -46,13 +46,12 @@ <h1 id="main_heading">My Questions</h1>
 
           <div id="panel_new" class="bb-tabpanel" role="tabpanel" aria-labelledby="tab1" aria-hidden="false">
             <section id="myquestions" class="questions all" data-all="true"></section>
-            <button id="load-more" class="button-gray hide">Load more…</button>
+            <button id="load-more-my-questions" data-container="myquestions" class="button-gray hide">Load more…</button>
           </div>
 
           <div id="panel_active" class="bb-tabpanel" role="tabpanel" aria-labelledby="tab2" aria-hidden="true">
-            <section data-type="buddyup-list">
-              <p>No active question threads.</p>
-            </section>
+            <section id="activequestions" class="questions active" data-type="buddyup-list"></section>
+            <button id="load-more-active-questions" data-container="activequestions" class="button-gray hide">Load more…</button>
```

Comment:
Rather than a data-container, I'd rather have a `class="js-question-container"` on the container. This is a bit more DRY because right now if you change the ID of the container, you have to change the data-container info.